### PR TITLE
chore: retrieve imported certificates from manifest and ssm

### PR DESCRIPTION
**Pull request by Lou1415926**
This PR reads the deployed environment's manifest before attempting reading from SSM configuration for environment for imported certs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by iamhopaul123 — APPROVED** [review]

File: internal/pkg/cli/deploy/deploy.go [hunk 1]
```diff
@@ -1267,15 +1298,19 @@ func (d *backendSvcDeployer) validateALBRuntime() error {
 	if d.backendMft.RoutingRule.IsEmpty() {
 		return nil
 	}
+	certsImported, err := d.envHasImportedCertificates()
```
Comment (efekarakus) [inline internal/pkg/cli/deploy/deploy.go]: nit:
```suggestion
	hasImportedCerts, err := d.envHasImportedCertificates()
```

File: internal/pkg/deploy/cloudformation/stack/lb_web_svc.go [hunk 2]
```diff
@@ -56,7 +56,7 @@ func WithNLB(cidrBlocks []string) func(s *LoadBalancedWebService) {
 // LoadBalancedWebServiceConfig contains fields to configure LoadBalancedWebService.
 type LoadBalancedWebServiceConfig struct {
 	App           *config.Application
-	Env           *config.Environment
+	EnvConfig     *manifest.Environment
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/stack/lb_web_svc.go]: nit maybe:
```suggestion
	EnvManifest     *manifest.Environment
```
`EnvConfig` makes me think of the `config` pkg

**Review by efekarakus — COMMENTED** [review]
Where is the "and ssm" part 😂 it seems to be strictly from the manifest, am I misunderstanding a flow?

**Comment by Lou1415926** [comment]
> Where is the "and ssm" part 😂 it seems to be strictly from the manifest, am I misunderstanding a flow?


🤣  because you've handled it here! https://github.com/aws/copilot-cli/blob/7aaef5c550892a858f99d21f6f8df7a433e2cbb4/internal/pkg/describe/env.go#L154

**Comment by iamhopaul123** [comment]
> 🤣 because you've handled it here!

blame efe karakus (1 month ago)

**Review by efekarakus — APPROVED** [review]

**Pull request merged**